lib/dbutils.js

```diff
@@ -88,6 +88,9 @@ dbu.conversions = {
     },
     uuid: {
         read: toString()
+    },
+    long: {
+        type: 'string'
```

Comment:
The default converter seems to simply return the value. Shouldn't we enforce toString() here?